Src/EB/AMReX_EB_STL_utils.cpp

```diff
@@ -0,0 +1,151 @@
+#include<AMReX_EB_STL_utils.H>
+#include<AMReX_EB_triGeomOps_K.H>
+
+//================================================================================
+void STLtools::read_ascii_stl_file(std::string fname)
+{
+    std::string tmpline,tmp1,tmp2;
+    int nlines=0;
+
+    std::ifstream infile(fname.c_str());
```

Comment:
All processes reading from disk will not scale well on supercomputers.  I suggest we use `ParallelDescriptor::ReadAndBcastFile` and then read from stringstream.  For an example of using this, see `AMReX_VisMF.cpp`.

Reply:
This is a neat trick! I have added it to the stl reading function